### PR TITLE
Add mnemonics for ldexp and fma

### PR DESCRIPTION
**Pull request by dneto0**
See #3567

### CODE REVIEW

File: wgsl/index.bs [hunk 1]
```diff
@@ -12252,6 +12263,10 @@ Note: The result is not mathematically meaningful if `e` &le; 0.
     </blockquote>                
 
     [=Component-wise=] when `T` is a vector.
+
+    Note: A mnemonic for the name `ldexp` is "load exponent".
+    The name may have been taken from the corresponding instruction in the floating point unit of
+    the PDP-11.
```
Comment (dneto0) [inline wgsl/index.bs:12269]: See how "ldexp" is mentioned in
"8.9 Floating point instructions" in the Assembler Reference Manual part of the [Unix Version 7 Volume 2B.](https://s3.amazonaws.com/plan9-bell-labs/7thEdMan/v7vol2b.pdf)

Comment (dneto0) [reply wgsl/index.bs:12269]: And Unix was famously developed on the PDP-11.
http://bitsavers.org/pdf/dec/pdp11/handbooks/PDP11_Handbook1979.pdf found from Wikipedia entry for PDP-11

[FP11-E floating point users guide](http://www.bitsavers.org/pdf/dec/pdp11/1160/EK-FP11E-UG-001_FP11-E_Users_Guide_Dec77.pdf) (1977) found by searching for "PDP-11 floating point instructions"

Comment (munrocket) [reply wgsl/index.bs:12269]: wow, so cool, didn't know this mnemonic about ldexp.

**Review by alan-baker — APPROVED** [review]

File: wgsl/index.bs (hunk 1 above)
Comment (dneto0) [reply wgsl/index.bs:12269]: yeah.  It didn't take too long to find.  But then that led me on a digging expedition for 'frexp'. I could not find an authoritative reference.

**Pull request merged**